src/alda/AldaClient.java

```diff
@@ -68,8 +73,9 @@ public static void updateAlda() throws URISyntaxException {
     }
 
     if (downloadURL == null) {
-      System.err.println("Alda download link not found for your platform.");
-      return;
+      throw new UnsuccessfulException(
+        "Alda download link not found for your platform."
```

Comment:
Maybe we should ask people to file an issue in this case? It should only happen if there's a broken release for some reason (I think).

Maybe we could even make this a property of the constructor of the alda exception (to add a 'file an issue' to this error).